script/pptemplate

```diff
@@ -117,6 +117,80 @@ sub postamble { ::pdlpp_postamble(\@pd_srcs) }
 EOM
 }
 
+sub pdManifest {
+  my ($pdname) = @_;
+  my $pd_dir = dirname $pdname;
+  return <<"END_OF_MANIFEST";
+$pdname
+Makefile.PL
+MANIFEST
+MANIFEST.SKIP
+t/basic.t
+END_OF_MANIFEST
+}
+
+sub pdManifest_skip {
+  my ($pdname,$dir) = @_;
+  my $pd_dir = dirname $pdname;
+  my $pm_name = $pdname =~ s/\.pd$/.pm/r;
+  return <<"SKIP_PROJECT" . <<'SKIP_GENERAL';
+# This project's temporary files
+$pm_name
+$pd_dir/.*-pp-.*
+$pd_dir/.*\\.bs\$
+$pd_dir/.*\\.c\$
```

Comment:
This is too broad, since the developer might add their own C files, for example https://github.com/PDLPorters/pdl/blob/master/lib/PDL/Primitive-xoshiro256plus.c - there's a reason I asked you to use the pattern `^lib/PDL/Compression(\.(pm|xs|c)$|-pp-)` :-)

Reply:
Ah, I seem to have missed that recommended pattern.  Was that on IRC? I'll fix that.